docs/csharp/nullable-references.md

```diff
@@ -380,6 +380,51 @@ public static class Program
 
 In the preceding example, the declaration of the array shows it holds non-nullable strings, while its elements are all initialized to `null`. Then, the variable `s` is assigned a `null` value (the first element of the array). Finally, the variable `s` is dereferenced causing a runtime exception.
 
+### Constructors
+
+Constructor of a class will still call the finalizer, even when there was an exception thrown by that constructor.
+<br/>The following example demonstrates that behavior:
+
+```csharp
+public class Foo
```

Comment:
Per our style guidelines, let's change the name of these types from "Foo" and "Bar".